editor/plugins/script_text_editor.cpp

```diff
@@ -1838,6 +1838,11 @@ void ScriptTextEditor::drop_data_fw(const Point2 &p_point, const Variant &p_data
 
 			if (drop_modifier_pressed && ResourceLoader::exists(path)) {
 				Ref<Resource> resource = ResourceLoader::load(path);
+				if (resource.is_null()) {
+					// Resource exists, but failed to load. We need only path and name, so we can use a dummy Resource instead.
+					resource.instantiate();
+					resource->set_path_cache(path);
```

Comment:
Would it make sense to always use this approach and save a costly `load`? Or it's not enough for some types of resources?

Reply:
`_get_dropped_resource_line()` also uses resource name, which currently can't be obtained without loading the Resource.